cves/CVE-2013-0305.yml

```diff
@@ -58,9 +67,11 @@ bounty:
   amt:
   announced:
   url:
-reviews: []
-bugs: []
-repo:
+reviews:
+bugs:
+- 1130445
+- 701186
+repo: https://github.com/django/django/commit/1f39eafd60761bf6a60b74d9e9859621da1b9363
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
```

Comment:
Some notes on the fix commits or additional information in another section would be great to understand how they addressed this vulnerability.